docs/guide/tutorials/commands.txt

```diff
@@ -1,3 +1,37 @@
 =================
 Database Commands
 =================
+
+Database commands allow you to perform a wide range of diagnostic and administrative
+tasks with the Node.js driver. For example, the
+:manual:`dbStats </reference/command/dbStats/>` command returns
+storage statistics for a given database. Use the ``command`` function to access
+database commands.
+
+.. code-block:: js
+
+   const { MongoClient } = require('mongodb');
+   const assert = require('assert');
```

Comment:
assert isn't used here. We should get rid of it

Reply:
agreed